wdio.conf.js

```diff
@@ -232,8 +240,12 @@ exports.config = {
      * @param {boolean} result.passed    true if test has passed, otherwise false
      * @param {object}  result.retries   information about spec related retries, e.g. `{ attempts: 0, limit: 0 }`
      */
-    // afterTest: function(test, context, { error, result, duration, passed, retries }) {
-    // },
+    afterTest: function(test, context, { error, result, duration, passed, retries }) {
+        const testName = test.title;
+                        Logger.info(`Test ${testName} is completed`)
+                        Logger.info("---------------------------------------------------------------")
+                browser.takeScreenshot();
```

Comment:
you don't need screenshot  for each test, just for failed 
also it would be great to log the result of the test with different log levels: info for passed, error for others